lib/main_containers.pm

```diff
@@ -130,6 +130,7 @@ sub load_host_tests_podman {
     load_3rd_party_image_test($run_args) unless is_staging;
     load_rt_workload($run_args) if is_rt;
     load_container_engine_privileged_mode($run_args);
+    loadtest 'containers/isolation', run_args => $run_args, name => $run_args->{runtime} . "_isolation" unless is_slem;
```

Comment:
Please add a comment why it's skipped on SLEM.

Reply:
Fixed.